patch-manager.tf

```diff
@@ -1,167 +1,167 @@
-resource "aws_ssm_maintenance_window" "scan" {
```

Comment:
Since there is no direct reference to the node groups, or simphera instance resources, try uncommenting this file.